membership/index.md

```diff
@@ -0,0 +1,45 @@
+---
+redirect_from: 
+    - membership/founding-membership
+---
+
+# Become a member
+
+We’re currently looking for members to help shape the Foundation for Public Code’s long term future.
```

Comment:
```suggestion
We are looking for members to help shape the Foundation for Public Code’s long term future.
```

Reply:
I generally go for contractions in our writing - what we're doing is complex enough that any stylistic friendliness and approachability is welcome.